agent/src/beerocks/slave/ap_manager_thread.cpp

```diff
@@ -1527,7 +1528,7 @@ void ap_manager_thread::handle_hostapd_attached()
             usleep(ACS_READ_SLEEP_USC);
         }
     } else {
-        ap_wlan_hal->read_supported_channels();
+        ap_wlan_hal->update_preference_channels_from_supported_channels();
```

Comment:
removed the `update_preference_channels_from_supported_channels`
as reqeusted by @tomereli
the `read_supported_channels()` was also changed to `read_preferred_channels` since that is its only use.

The supported channels are stored as part of the `refresh_radio_info()`